package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eppo/react-native-sdk",
-  "version": "3.5.1",
+  "version": "3.5.2",
```

Comment:
Thoughts on doing 3.4.1 so we're not skipping a minor version? I'm also fine with this

Reply:
We would need to apply this fix in a branch from 3.4.0 to avoid shipping the minor change brought by 3.5 along with it.
I think it's not uncommon to skip minor versions - we have done so in the past when we've merged new features so quickly they get merged into a single release

Reply:
Sounds good. Yep let's proceed with the skip